src/dataflow-types/src/plan/reduce.rs

```diff
@@ -739,7 +771,6 @@ impl ReducePlan {
                         buckets.push(current as u64);
                         current *= 16;
```

Comment:
It seems like the fix should be to make this `saturating_mul`?

Reply:
Indeed it should be! I just forgot on Friday that `saturating_mul` existed. Thanks for pointing this out!